frontend/src/components/application/editAppFilingName/actions.ts

```diff
@@ -0,0 +1,47 @@
+"use server";
+
+import { getSession } from "src/services/auth/session";
+import { updateApplicationFilingName } from "src/services/fetch/fetchers/applicationFetcher";
+
+import { revalidateTag } from "next/cache";
+
+export type UpdateAppFilingNameActionState =
+  | {
+      success: boolean;
+      error: string | null;
+    }
+  | undefined;
+
+export const updateAppFilingNameAction = async (
+  _prevState: UpdateAppFilingNameActionState | undefined,
+  formData: FormData,
+): Promise<UpdateAppFilingNameActionState> => {
+  const applicationId = formData.get("application_id") as string;
+  const applicationName = formData.get("application_name") as string;
+  const session = await getSession();
+
+  if (!session || !session.token) {
```

Comment:
[nit] can we move the early return up as far as we can (above the logic for pulling data out of the form)?